olp-cpp-sdk-dataservice-write/src/StreamLayerClientImpl.cpp

```diff
@@ -34,16 +35,9 @@
 #include "generated/ConfigApi.h"
 #include "generated/IngestApi.h"
 #include "generated/PublishApi.h"
-
-// clang-format off
-#include <generated/serializer/PublishDataRequestSerializer.h>
-#include <generated/serializer/JsonSerializer.h>
-// clang-format on
-
-// clang-format off
-#include <generated/parser/PublishDataRequestParser.h>
-#include <olp/core/generated/parser/JsonParser.h>
-// clang-format on
```

Comment:
This order and clang-format off is necessary to be able to compile, as one defines the to_json method and the other the serializer that uses this to_json()

Reply:
Done